src/localization/locales/en-US.json

```diff
@@ -365,7 +366,8 @@
         "form_profile_partnership_status": "Is a landscape partnership managing this landscape?",
         "profile_profile_card_partnership_status_yes": "Yes",
         "profile_profile_card_partnership_status_in_progress": "In progress",
-         "profile_profile_card_partnership_status_no": "No",
+        "profile_profile_card_partnership_status_no": "No",
+        "profile_affiliation_card_empty": "[TODO] No affiliation content",
```

Comment:
question: What is the context for this? Does it make sense to omit the parent component instead of displaying an error?